.travis.yml

```diff
@@ -25,11 +25,21 @@ script:
   - docker images
   - docker run `make version` --help
 
-# push an image on tags
+# push images to DockerHub on tags
 deploy:
   provider: script
   script: bash docker_deploy.sh
   skip_cleanup: true
   on:
     tags: true
     condition: -n "$DOCKER_PASSWORD"
+
+# push packages to PyPI on tags
+deploy:
```

Comment:
Is it legal to have duplicate keys ? It doesn't look like valid YAML.

Reply:
I think that we might want this https://docs.travis-ci.com/user/deployment#deploying-to-multiple-providers